src/common/types/sdk-options.types.ts

```diff
@@ -1,4 +1,4 @@
-import type { Locale, Offset } from '../../services/video-conference-manager/types';
+import type { Locale, Avatar, Offset } from '../../services/video-conference-manager/types';
```

Comment:
You should use the default interface for avatars. And remove this import.


```suggestion
import type { Locale, Offset } from '../../services/video-conference-manager/types';
```